StudyBuddyApp/StudyBuddyApp/Views/HelpRequestListPage.xaml.cs

```diff
@@ -12,28 +12,41 @@
 using StudyBuddyApp.Models;
 using StudyBuddyApp.ViewModels;
 using StudyBuddyShared.Utility.Extensions;
+using System.Collections.Generic;
 
 namespace StudyBuddyApp.Views
 {
     [XamlCompilation(XamlCompilationOptions.Compile)]
     public partial class HelpRequestListPage : ContentPage
     {
         public ObservableCollection<HelpRequestModel> Items { get; set; }
-
+        public ObservableCollection<HelpRequestModel> FilteredItems { get; set; }
         public HelpRequestListPage()
         {
             InitializeComponent();
 
             Items = new ObservableCollection<HelpRequestModel>
             {};
+            FilteredItems = new ObservableCollection<HelpRequestModel>
+            { };
+
             HelpRequestListGetter();
+            Filter(null, null, false);
 
-            HelpRequestList.ItemsSource = Items;
+            HelpRequestList.ItemsSource = FilteredItems;
+            PickCategory.ItemsSource = Items;
```

Comment:
Kaip supratau čia paimi iš visų HelpRequest kategorijas, iš to gaunasi, kad pasirinkimų sąraše dubliuojasi daug kategorijų.
Tiesiog su getteriu pasiimk kategorijas iš interneto